onnxruntime/python/tools/transformers/convert_generation.py

```diff
@@ -1709,6 +1722,10 @@
         )
         graph_inputs.append(presence_mask)
 
+    if is_sampling and args.seed:
+        seed = onnx.helper.make_tensor_value_info("seed", TensorProto.INT32, [1])
```

Comment:
>seed [](http://example.com/codeflow?start=51&length=4)

How about to make seed configurable? default set to -1. if >0, use the seed passed in by args

Reply:
currently we only test sampling with default seed and small top_p. so this flag limits to help create the model with that input